src/domdiv/main.py

```diff
@@ -1498,6 +1499,24 @@ def __init__(self, order, baseCards):
         for tag in baseCards:
             self.baseCards.append(baseCards[tag])
 
+        # Set the locale to the selected language. Necessary for correct sorting using accented characters
+        # "nt" = Windows
+        # "posix" = Linux / macOS
+        try:
+            if os.name == "nt":
+                locale.setlocale(locale.LC_COLLATE, lang)
+            elif os.name == "posix":
+                locale.setlocale(locale.LC_COLLATE, (lang, ""))
```

Comment:
Can you try specifying a `UTF-8` encoding here?

Reply:
Done, tested on Ubuntu, works fine. After some reading, defaulting to UTF-8 seems to be a good idea.